rust/flatbuffers/src/vector.rs

```diff
@@ -52,6 +52,9 @@ impl<'a, T: Follow<'a> + 'a> Vector<'a, T> {
         debug_assert!(sz > 0);
         T::follow(self.0, self.1 as usize + SIZE_UOFFSET + sz * idx)
     }
+
+    #[inline(always)]
```

Comment:
Let's just use `#[inline]` here -- it's more idiomatic and gives the compiler more flexibility.

Reply:
Understood.